source/features/sync-pr-commit-title.tsx

```diff
@@ -24,52 +28,43 @@ function createCommitTitle(): string {
 }
 
 function needsSubmission(): boolean {
-	const inputField = getCommitTitleField();
-	if (!inputField || inputField.value === '') {
-		return false;
-	}
-
-	// Ensure that the required fields are on the page
-	if (!select.exists(prTitleFieldSelector + ',' + prTitleSubmitSelector)) {
-		features.log.error(import.meta.url, 'Can’t update the PR title');
-		return false;
-	}
-
-	return createCommitTitle() !== inputField.value;
+	const currentCommitTitle = getCurrentCommitTitle();
+	return Boolean(currentCommitTitle) && (createCommitTitle() !== currentCommitTitle);
 }
 
 function getUI(): HTMLElement {
-	return select(`${mergeFormSelector} .rgh-sync-pr-commit-title-note`) ?? (
+	const cancelButton = <button type="button" className="btn-link Link--muted text-underline rgh-sync-pr-commit-title">Cancel</button>;
+	return select('.rgh-sync-pr-commit-title-note') ?? (
 		<p className="note rgh-sync-pr-commit-title-note">
-			The title of this PR will be updated to match this title. <button type="button" className="btn-link Link--muted text-underline rgh-sync-pr-commit-title">Cancel</button>
+			The title of this PR will be updated to match this title. {cancelButton}
 		</p>
 	);
 }
 
 function updateUI(): void {
 	if (needsSubmission()) {
-		getCommitTitleField()!.after(getUI());
+		getCurrentCommitTitleField()!.after(getUI());
 	} else {
 		getUI().remove();
 	}
 }
 
-function updatePRTitle(): void {
+async function updatePRTitle(): Promise<void> {
 	if (!needsSubmission()) {
 		return;
 	}
 
 	// Remove PR number from commit title
-	const prTitle = getCommitTitleField()!.value
-		.replace(regexJoin(/\s*\(/, '#' + getConversationNumber()!, /\)$/), '');
+	const title = cleanPrCommitTitle(getCurrentCommitTitle()!, getConversationNumber()!);
 
-	// Fill and submit title-change form
-	select(prTitleFieldSelector)!.value = prTitle;
-	select(prTitleSubmitSelector)!.click(); // `form.submit()` isn't sent via ajax
+	await api.v3(`pulls/${getConversationNumber()!}`, {
+		method: 'PATCH',
+		body: {title},
+	});
```

Comment:
Tested, it works https://github.com/refined-github/sandbox/pull/62

<img width="525" alt="Screenshot 10" src="https://user-images.githubusercontent.com/1402241/234368877-715822a0-1145-497f-ad1d-4f6caceb6ec4.png">


<img width="384" alt="Screenshot 11" src="https://user-images.githubusercontent.com/1402241/234368859-6550e134-c40a-4c6b-abae-e16761501a2c.png">